docs/architecture/adr-005-qgb-reduce-state-usage.md

```diff
@@ -40,7 +44,7 @@ However, slashing will be very difficult, especially for liveness, i.e. an orche
 
 Remove the `MsgValsetConfirm` defined in [here](https://github.com/celestiaorg/celestia-app/blob/a965914b8a467f0384b17d9a8a0bb1ac62f384db/proto/qgb/msgs.proto#L24-L49)
 And also, the `MsgDataCommitmentConfirm` defined in [here](
-https://github.com/celestiaorg/celestia-app/blob/a965914b8a467f0384b17d9a8a0bb1ac62f384db/proto/qgb/msgs.proto#L55-L76).
+<https://github.com/celestiaorg/celestia-app/blob/a965914b8a467f0384b17d9a8a0bb1ac62f384db/proto/qgb/msgs.proto#L55-L76>).
```

Comment:
[Question][Non-blocking]

What's the difference between `[...](<link>)` and `[...](link)`?

Reply:
there isn't a difference. My editor auto surrounded the link with `<` and `>` likely to conform to https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md but in this case, it's not a bare URL because of the preceding `[here]`